modules/installation-special-config-butane.adoc

```diff
@@ -0,0 +1,116 @@
+https://coreos.github.io/butane/config-openshift-v4_8/[OpenShift Butane config spec].
+
+[id="rhcos-installing-butane_{context}"]
+== Installing Butane
+
+You can install the Butane tool (`butane`) to create {product-title} machine configs from a command-line interface. You can install `butane` on Linux, Windows, or macOS by downloading the related binary file.
+
+[TIP]
+====
+Butane releases are backwards-compatible with older releases and with the Fedora CoreOS Config Transpiler (FCCT).
+====
+
+.Procedure
+
+. Navigate to the Butane image download page at https://mirror.openshift.com/pub/openshift-v4/clients/butane/.
+. Get the `butane` binary:
+.. For the newest version of Butane, save the latest `butane` image to your current directory:
++
+[source,terminal]
+----
+$ curl https://mirror.openshift.com/pub/openshift-v4/clients/butane/latest/butane --output butane
+----
++
+.. Optional: For a specific type of architecture you are installing Butane on, such as aarch64 or ppc641e, indicate the appropriate URL. For example:
++
+[source,terminal]
+----
+$ curl https://mirror.openshift.com/pub/openshift-v4/clients/butane/latest/butane-aarch64
+----
+
+. Move the `butane` binary file to a directory on your `PATH`.
++
+To check your `PATH`, open a terminal and execute the following command:
++
+[source,terminal]
+----
+$ echo $PATH
+----
++
+. Make the downloaded binary file (for example, `butane`) executable:
++
+[source,terminal]
+----
+$ chmod +x butane
+----
++
+You can now use the Butane tool by running the `butane` command:
++
+[source,terminal]
+----
+$ butane <butane_file.bu>
+----
+
+[id="rhcos-creating-with-butane_{context}"]
+== Creating a `MachineConfig` object by using Butane
+
+By using Butane, you can produce a `MachineConfig` object so that you can configure worker or master nodes at installation time or via the Machine Config Operator.
+
+.Prerequisites
+
+* You have installed the `butane` utility.
+
+.Procedure
+
+. Create a Butane config YAML file. The following example tells the {product-title} cluster to use a real-time kernel for all worker nodes:
++
+[source,terminal]
+----
+$ cat <<EOF > 99-worker-realtime.bu
+variant: openshift
+version: 4.8.0
+metadata:
+  name: 99-worker-realtime
+  labels:
+    machineconfiguration.openshift.io/role: worker
+openshift:
+    kernel_type: realtime
+EOF
+----
++
+[NOTE]
+====
+The `99-worker-realtime.bu` is set to create a machine config for the real-time kernel on worker nodes. To deploy on control plane nodes, change the role from `worker` to `master` and change the kernel type to `default`. To do both, you could repeat the whole procedure using different file names for the two types of deployments.
+====
+
+. Create a `MachineConfig` object by giving Butane the file that you created in the previous step, for example:
++
+[source,terminal]
+----
+$ butane $HOME/<cluster_config_dir>/99-worker-realtime.bu -o ./99-worker-realtime.yaml
```

Comment:
We didn't tell them where to put the Butane config, so maybe we should omit the directory part of the input argument?

Reply:
Yes, agreed